php/WP_CLI/DocParser.php

```diff
@@ -70,6 +70,23 @@ public function get_longdesc() {
 				break;
 			}
 
+			if ( preg_match_all( '/(\[.+?\]\((https?:\/\/.+?)\))/', $line, $m ) ) {
```

Comment:
Could we get a plain `phpunit` test for this too? There's an existing `TestDocParser` class you can add to.